src/table/table-body.jsx

```diff
@@ -419,9 +415,14 @@ const TableBody = React.createClass({
     const rows = this._createRows();
 
     return (
-      <tbody className={className} style={prepareStyles(Object.assign({}, style))}>
+      <ClickAwayListener
+        className={className}
+        onClickAway={this.componentClickAway}
+        style={prepareStyles(Object.assign({}, style))}
+        type="tbody"
```

Comment:
This is a regression. Right now the `ClickAwayListener` has an `EventListener` that renders a `<noscript/>` because it is returning null. This breaks validation because tbody cannot have that type of tag.

This was fixed in React here: https://github.com/facebook/react/pull/5451

We'll probably have to find another way for now.